doc/man3/EVP_DigestInit.pod

```diff
@@ -208,6 +208,10 @@ value explicitly fetched with EVP_MD_fetch().
 If I<impl> is non-NULL, its implementation of the digest I<type> is used if
 there is one, and if not, the default implementation is used.
 
+If EVP_DigestInit_ex() is called on I<ctx> already initialized
```

Comment:
Maybe scramble around a bit?? I will leave it up to you..

I<type> can be NULL if I<ctx> has already been initialized via a previous call to EVP_DigestInit_ex()